### PR TITLE
fixed local checks

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

The PR includes local-check script and remove the functions which according to the `golangci-lint` checks, you can run `make local-check` command and to validate the code

### CODE REVIEW

File: linkerd/linkerd.go [hunk 1]
```diff
@@ -700,8 +552,8 @@ func (iClient *Client) applyConfigChange(ctx context.Context, deploymentYAML, na
 		}
 		err := iClient.k8sDynamicClient.Resource(mappingNamespace.Resource).Delete(namespace, deleteOptions)
 		if err != nil {
-			// logrus.Error(fmt.Sprintf("Delete the %s %s failed", dataNamespace.GetObjectKind().GroupVersionKind().Kind, namespace))
-			// return err
+			logrus.Error(fmt.Sprintf("Delete the %s %s failed", dataNamespace.GetObjectKind().GroupVersionKind().Kind, namespace))
```
Comment (Aisuko) [inline linkerd/linkerd.go:555]: I saw we command out the code, Is that mean that we do not need to return the error and output the logs?

**Comment by Aisuko** [comment]
@leecalcote `2020-10-16T03:53:54.1684867Z Error parsing reference: "***:edge-refs/pull/130/merge" is not a valid repository/tag: invalid reference format`. The `docker tag ${{ secrets.IMAGE_NAME }}:edge-latest ${{ secrets.IMAGE_NAME }}:edge-${GITHUB_REF/refs\/tags\//}` was not work very well.

**Comment by Aisuko** [comment]
> @leecalcote Is there any idea? I saw that work for other projects
> 
> `2020-10-16T03:53:54.1684867Z Error parsing reference: "***:edge-refs/pull/130/merge" is not a valid repository/tag: invalid reference format`.
> The `docker tag ${{ secrets.IMAGE_NAME }}:edge-latest ${{ secrets.IMAGE_NAME }}:edge-${GITHUB_REF/refs\/tags\//}` was not work very well.

Oh, that the when we were in the edge situation we do not have any tag, so, we should remove the tag. And if we push with a tag, it's must be we are going to release the stable versions.

File: linkerd/linkerd.go (hunk 1 above)
Comment (kumarabd) [reply linkerd/linkerd.go:553]: @Aisuko that needs to be corrected. Basically yes we do want to delete, but only if it exists.

Comment (Aisuko) [reply linkerd/linkerd.go:553]: If I understand you in a right way that we want to ignore the `err` right?

**Comment by Aisuko** [comment]
We can merge this I believe @kumarabd

**Review by kumarabd — APPROVED** [review]

**Comment by kumarabd** [comment]
> We can merge this I believe @kumarabd

@Aisuko Sounds good. We will address to the tags later on.

**Pull request merged**